pkg/providers/snow/objects.go

```diff
@@ -31,7 +40,7 @@ func ControlPlaneObjects(ctx context.Context, clusterSpec *cluster.Spec, kubeCli
 	}
 	capiCluster := CAPICluster(clusterSpec, snowCluster, kubeadmControlPlane)
 
-	return []kubernetes.Object{capiCluster, snowCluster, kubeadmControlPlane, new}, nil
+	return []kubernetes.Object{capiCluster, snowCluster, kubeadmControlPlane, new, clusterSpec.SnowCredentialsSecret, capasCredentialsSecret}, nil
```

Comment:
interesting, why do we return `clusterSpec.SnowCredentialsSecret` from here?
I would expect the CLI to handle this independently. For example, when applying the cluster spec (since this secret now is part of the snow eks-a cluster specification)